code/modules/mapping/modular_map_loader/modular_map_loader.dm

```diff
@@ -11,7 +11,7 @@
 	/// Key used to look up the appropriate map paths in the associated .toml file
 	var/key = null
 
-/obj/modular_map_root/Initialize(mapload)
+/obj/modular_map_root/New()
```

Comment:
You should keep it as `Initialize(mapload)` and instead use the `INITIALIZE_IMMEDIATE` macro